telemetry_core/src/main/java/com/newrelic/telemetry/metrics/MetricBuffer.java

```diff
@@ -47,6 +47,14 @@ public void addMetric(Metric metric) {
     metrics.add(metric);
   }
 
+  /**
+   * Get the size of the {@link Metric Metrics} buffer.
+   *
+   * @return Size of the {@link Metric Metrics} buffer.
+   */
+  public int size() {
```

Comment:
Can you add a unit test to verify this works? Thanks!

Reply:
@jkwatson Where to add unit testing.  the repo contains only an integration test, do I have to add a new test into the integration_test folder suite ?

Reply:
The repo contains many many tests in the src/test directory. Please add a test case to the `MetricBufferTest` for this change.